www/scripts/codecheckerviewer/ListOfRuns.js

```diff
@@ -247,7 +247,7 @@ function (declare, dom, ItemFileWriteStore, topic, Dialog, Button,
           var runNameFilter = this.get('value');
           this.timer = setTimeout(function () {
             var runFilter = new CC_OBJECTS.RunFilter();
-            runFilter.names = [runNameFilter];
+            runFilter.names = [runNameFilter + '*'];
```

Comment:
What it we would use `['*' + runNameFilter + '*']` so you could type partial run names?

Reply:
I think if the user would like to search partially, he should write `*` quantifier to the input box. This is how the left side filters works on the Bug overview page (like file filter).

Reply:
Ok, let's be consistent for every regex filter.